controllers/keysend.ctrl.go

```diff
@@ -91,6 +93,10 @@ func (controller *KeySendController) KeySend(c echo.Context) error {
 	if err != nil {
 		return err
 	}
+	if _, err := hex.DecodeString(invoice.DestinationPubkeyHex); err != nil || len(invoice.DestinationPubkeyHex) != common.DestinationPubkeyHexSize {
+		c.Logger().Errorf("Invalid destination pubkey hex user_id:%v pubkey:%v", userID, len(invoice.DestinationPubkeyHex))
+		return c.JSON(http.StatusBadRequest, responses.BadArgumentsError)
```

Comment:
I would also add a specific "Invalid destination pubkey" error for this and use that one in the responses here.